packages/core/data-transfer/src/strapi/remote/handlers/pull.ts

```diff
@@ -167,6 +180,20 @@ export const createPullController = handlerControllerFactory<Partial<PullHandler
     const stream = this.streams?.[stage];
 
     const batchLength = () => Buffer.byteLength(JSON.stringify(batch));
+
+    const maybeConfirm = async (data: any) => {
+      try {
+        await this.confirm(data);
+      } catch (error) {
+        // Handle the error, log it, or take other appropriate actions
+
+        strapi?.log.error(
```

Comment:
Does this need to be optional like this?
```suggestion
        strapi.log.error(
```

Reply:
I wanted to be safe in case it's used in some context where it's not globally defined, but yeah, in the typical case it works